lib/active_record/session_store/session.rb

```diff
@@ -71,6 +71,21 @@ def loaded?
         @data
       end
 
+      def secure!
```

Comment:
Why this method was added?

Reply:
I guess this should probably be documented.

Normally, sessions are only upgraded to secure sessions, when a user makes a request using the insecure session. This allows existing user sessions to be upgraded to secure sessions while the users are offline.

Reply:
This will allow maintainers to create a migration which basically performs a `ActionDispatch::Session::ActiveRecordStore.session_class.find_each(&:secure!)` to migrate all existing insecure sessions to secure ones. The next lookup with the `private_id` will find the secured value in the DB and just work as for new secure sessions. The fallback case won't be executed and no additional DB query will be performed (faster/lower load).